sdk/typescript/test/e2e/object-vector.test.ts

```diff
@@ -66,5 +68,21 @@ describe.each([{ useLocalTxnBuilder: true }, { useLocalTxnBuilder: false }])(
     it('Test object vector', async () => {
       await destroyObjects([await mintObject(7), await mintObject(42)]);
     });
+
+    it('Test regular arg mixed with object vector arg', async () => {
```

Comment:
not blocking but would be great to have another test with another argument after the object vector